Cargo.toml

```diff
@@ -30,6 +30,10 @@ rstest = "0.15.0"
 [features]
 graphviz-dot = ["dot-writer"]
 
+[[bench]]
+name = "calibration"
```

Comment:
This is still benching the `parser`, it's just an additional fixture, can we include it in the same command?